src/templates/uk/2019/contributors.html

```diff
@@ -2,9 +2,9 @@
 
 {% block title %}Контриб'ютори 2019 року | Web Almanac від HTTP Archive{% endblock %}
 
-{% block description %}{{ config.contributors.items() | length }} неймовірних людей, які долучилися до створення Web Almanac 2019 як аналітики, автори, дизайнери, розробники, рецензенти, керівники, редактори та перекладачі.{% endblock %}
+{% block description %}{{ config.contributors.items() | length }} {{ plural_ru(config.contributors.items() | length | int, ["неймовірна людина", "неймовірні людини", "неймовірних людей"]) }}, які долучилися до створення Web Almanac 2019 як аналітики, автори, дизайнери, розробники, рецензенти, керівники, редактори та перекладачі.{% endblock %}
 
-{% block filter_by_team %}Фільтрувати за роллю в команді: <span id="filtered-contributors">{{ self.contributors() }}</span><span id="contributors-total-text" class="hidden"> з <span id="contributors-total">{{ config.contributors.items() | length }}</span></span> контриб'юторів.{% endblock %}
+{% block filter_by_team %}Фільтрувати за роллю в команді: <span id="filtered-contributors">{{ self.contributors() }}</span><span id="contributors-total-text" class="hidden"> з <span id="contributors-total">{{ config.contributors.items() | length }}</span></span> {{ plural_ru(config.contributors.items() | length | int, ["контриб'ютор", "контриб'ютори", "контриб'юторів"]) }}.{% endblock %}
```

Comment:
You missed a couple, @tymosh

Reply:
Resolving this as code has changed so let's pick up on latest code by doing another review (and ideally suggesting the changes using the ![suggest](https://user-images.githubusercontent.com/10931297/106916108-b89dae80-66fe-11eb-8204-2b3592a29180.png) icon for my poor, non-Russian/non-Ukranian brain!).